src/models/global.rs

```diff
@@ -147,23 +169,74 @@ impl Default for V4L2Standard {
     }
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "lowercase", deny_unknown_fields)]
+pub enum FlipMode {
+    #[serde(rename = "NO_CHANGE")]
+    NoChange,
+    Horizontal,
+    Vertical,
+    Both,
+}
+
+impl Default for FlipMode {
+    fn default() -> Self {
+        Self::NoChange
+    }
+}
+
+#[derive(Debug, Clone, Copy, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "UPPERCASE", deny_unknown_fields)]
+pub enum Pixelformat {
```

Comment:
Should be `PixelFormat` with an uppercase `F`, to stay consistent

Reply:
Ok sure